packages/node_modules/@node-red/runtime/lib/nodes/Node.js

```diff
@@ -236,6 +239,22 @@ Node.prototype.receive = function(msg) {
     }
 };
 
+Node.prototype.getenv = function(name) {
```

Comment:
I don't currently see why any changes to the `Node` prototype are needed. This feature is specific to Subflows - not Nodes in general.

Reply:
Removed from updated code.